official/mnist/mnist.py

```diff
@@ -37,6 +45,7 @@ def __init__(self, data_format):
         typically faster on CPUs. See
         https://www.tensorflow.org/performance/performance_guide#data_formats
     """
+    super(Model, self).__init__()
```

Comment:
Hey @asimshankar, this line causes an error for me when I run the model. Specifically:

```
  File "mnist/mnist.py", line 48, in __init__
    super(Model, self).__init__()
TypeError: __init__() takes at least 3 arguments (1 given)
```

Note I'm currently on TF 1.5. Is this a bug, or is it because this code only works with TF 1.6? If so we should update the README at https://github.com/tensorflow/models/tree/master/official.

Reply:
@nealwu : It will infact require version 1.7. Perhaps we should just remove the line

"If you are on a version of TensorFlow earlier than 1.4, please update your installation."
from the README? Since just before that it talks about tagged branches.

Reply:
That line is fine. I was referring to this one specifically:

"We aim to keep them backwards compatible with the latest release when possible (currently TensorFlow 1.5), but we cannot always guarantee compatibility."

If there is also a way we can keep this model backwards-compatible with 1.5/1.6 until 1.7 is the latest release, I would be strongly in favor of that.